seedlet/assets/sass/blocks/utilities/_style.scss

```diff
@@ -38,13 +41,16 @@
 	margin-bottom: var(--global--spacing-vertical);
 	/*rtl:ignore*/
 	margin-left: var(--global--spacing-horizontal);
+}
+
+.entry-content > .alignright {
```

Comment:
I spoke / committed too soon. I think using need to use `.entry-content > .alignleft/right`. Otherwise we'd have to use a lot of `:not` selectors to determine when to apply the `%responsive-alignleft/right-width`s.